packages/fxa-auth-server/.storybook/main.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  stories: ['../lib/senders/emails/**/*.stories.tsx'],
+  addons: ['@storybook/addon-docs', '@storybook/addon-controls'],
```

Comment:
Added these addons so as to implement things stated in [this comment](https://github.com/mozilla/fxa/pull/8765#issuecomment-862794694)